dev/src/codewind/project/ProjectType.ts

```diff
@@ -47,6 +47,9 @@ export class ProjectType {
         if (internalType === this.InternalTypes.MICROPROFILE) {
             return ProjectType.Types.MICROPROFILE;
         }
+        else if (internalType === this.InternalTypes.APPSODY) {
```

Comment:
we should not need to treat the appsody type specially, or add this product code to afford testing, is there another way to do this?

Reply:
can we keep using the EXTENSION type?

Reply:
ya we can keep it as Extension type by giving extension name so we don't have to add this code in